### PR TITLE
feat: LT-1311 - Template decisions

### PR DESCRIPTION
**Pull request by andytaylorkainos**

### CODE REVIEW

File: templates/letter_templates/letter_template.html [hunk 1]
```diff
@@ -27,7 +27,7 @@ <h4 class="govuk-heading-s">{% lcs 'LetterTemplates.LetterTemplate.LAYOUT_TITLE'
 				<div class="govuk-grid-column-one-third">
 					<h4 class="govuk-heading-s">{% lcs 'LetterTemplates.LetterTemplate.RESTRICTED_TITLE' %}</h4>
 					<p class="govuk-label" id="case_types">
-						{% for case_type in letter_template.case_types %}{% if not forloop.first %}{% if forloop.last %} and {% else %}, {% endif %}{% endif %}{{ case_type.reference.value }}{% endfor %}
+						{% for case_type in letter_template.case_types %}{% if not forloop.first %}{% if forloop.last %} and {% else %}, {% endif %}{% endif %}{{ case_type.reference.key|upper }}{% endfor %}
```
Comment (BenAndrew-Kainos) [inline templates/letter_templates/letter_template.html]: Leave a comment please

Comment (andytaylorkainos) [reply templates/letter_templates/letter_template.html]: i feel like the comment is a little redundant here, its literally just using a capitalized version of the key (as the code implies)

**Comment by andytaylorkainos** [comment]
> Also could you show the list of decision types in the template header page

nicely spotted, done!

File: ui_automation_tests/features/letter_templates.feature [hunk 2]
```diff
@@ -4,7 +4,7 @@ As a logged in government user
 I want to configure standard templates for letter generation
 So that I can create standard letters used by case workers and they can be updated when required
 
-  @LT_1029_add_template @smoke
+  @LT_1029_add_template @smoke @AT
```
Comment (BenAndrew-Kainos) [inline ui_automation_tests/features/letter_templates.feature]: Remove @at

**Pull request merged**